CHANGELOG.md

```diff
@@ -26,7 +28,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## [1.0.3] - 2020-10-20
 
 ### Added
-
+telephone
```

Comment:
```suggestion
- Added telephone feature to PyTorch Lightning
```

Reply:
this is just a joke xD

But I would also add the deprecation of filepath to the changelog :)